Assets/Scripts/OpenDoor.cs

```diff
@@ -15,16 +15,15 @@ public class OpenDoor : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        Animator buttonAnim = button.gameObject.GetComponent<Animator>();
```

Comment:
Poprzez dodanie typu (`Animator` w tym przypadku), tworzymy nową zmienną lokalną o nazwie `buttonAnim`, zamiast nadpisywać już istniejącą `this.buttonAnim`.